### PR TITLE
Dialog implementation

### PR DESCRIPTION
**Pull request by Jinpeiqi**

### CODE REVIEW

**Comment by Jinpeiqi** [comment]
@rong4188

**Comment by Jinpeiqi** [comment]
@rong4188

File: MeetU/MeetU/Scripts/Controllers/Meetup/index.js [hunk 1]
```diff
@@ -141,39 +141,42 @@
...
-                            var meetupViews = $scope.meetupViews;
-                            meetupViews.splice(meetupViews.findIndex(function (meetup) {
-                                return meetup.meetup.id === meetupId;
-                            }), 1);
-                        }, function (message) {
-                            $log.error(message);
-                        });
-                }
-            };
+                $confirm({ text: 'Are you sure you want to delete?', title: 'Delete it', ok: 'Yes', cancel: 'No' })
```
Comment (rong4188) [inline MeetU/MeetU/Scripts/Controllers/Meetup/index.js]: can you please change the title to "Deleting a meetup" and the text to "Are you sure you want to delete this MeetUp?"

So does the comment deletion.

Comment (Jinpeiqi) [inline MeetU/MeetU/Scripts/Controllers/Meetup/index.js]: @rong4188 done

**Pull request merged**